packages/flutter/lib/logging.dart

```diff
@@ -0,0 +1,160 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
Instead of adding this as a new library in `lib/` (which all seem to just be export libraries), we should likely add this to one of the existing larger libraries (like the foundation one). So, logging.dart would show up in `lib/sec/foundation/`, and we'd export it from the `lib/foundation.dart` library.